### ISSUE TITLE
Alexander in TOAU 44 uses Draw-in immediately when it spawns

### ISSUE THREAD

**Issue by Goutan** [EdenServer/community#5021]
### Checklist

- [ x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x ] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [ x] I have provided my client version in the details. (type /ver into your game window)
- [x ] ***I have provided proof of my claim (era vide
![Alex Draw in](https://user-images.githubusercontent.com/62845606/165805207-c3bf0e7a-0f2d-4e2e-9271-2ae53e4a0005.jpg)
os, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Version: 30181205_0

When the cutscene ends between the phases and Alex spawns, it immediately begins drawing people in. It seems that we exited the cutscene on top of him instead of back at the entrance -> then our characters very quickly ran back to the entrance.

Looking at videos from retail (not era), it appears that people exited the cutscene near the door and did not get drawn in immediately.

https://youtu.be/TYUsOgOHomg?t=351

Image shows that we hadn't generated threat and the character who got drawn in hadn't moved since exiting the CS.

**Comment by KnowOne134** [comment]
Yes the cs supose to set you back to entrance. Since it didn't you all got aggro when it spawned at you

**Comment by KnowOne134** [comment]
tested and cs exited me at the correct spot, perhaps someone in PT was running 60 fps and had to exit the cs early?

has anyone else had this issue?

**Comment by Goutan** [comment]
The character who was drawn in first had an unlocked framerate during the CS

**Comment by KnowOne134** [comment]
Then yes I believe this is the issue. These toau mission cs's do not like running over 30
